### PR TITLE
Add tooltip to display username on hover over avatar

### LINKED ISSUE #96: Tooltip for Homepage-Avatars
When hovering a Avatar, the username of the hubber could be displayed with a tooltip

### PR DESCRIPTION
**Pull request by Sejal-16**
When user hovers over avatar on home page , the username is displayed with tooltip. 
image

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/371"></a>

Resolve #96

### CODE REVIEW

File: src/Components/Home.js [hunk 1]
```diff
@@ -24,13 +25,16 @@ function Home() {
...
+          <div className = "avatar-div" data-tip= {user.username} data-for='toolTip1' data-place='bottom'>
+            <Avatar
+              image={user.avatar}
+              shape="circle"
+              size="xlarge"
+              className="p-m-2"
+              imageAlt={user.username}
+            />
+          </div>
+          <ReactTooltip id="toolTip1" />
```
Comment (schmelto) [inline src/Components/Home.js:37]: I would use a more descriptive name here

Comment (Sejal-16) [reply src/Components/Home.js:37]: you mean id ? Maybe you can suggest a name

Comment (schmelto) [reply src/Components/Home.js:37]: puhhh maybe `toolTipUserName` but I'm open for other suggestions

File: package.json [hunk 2]
```diff
@@ -14,6 +14,7 @@
     "react-dom": "^17.0.2",
     "react-router-dom": "^5.3.0",
     "react-scripts": "4.0.3",
+    "react-tooltip": "^4.2.21",
```
Comment (Cahllagerfeld) [inline package.json:17]: [PrimeReact](https://primefaces.org/primereact/showcase/#/tooltip) already provides a Tooltip-Function, so we dont need an additional package for this.

**Review by Cahllagerfeld — CHANGES_REQUESTED** [review]
Just a small comment 👍

File: package.json (hunk 2 above)
Comment (Sejal-16) [reply package.json:17]: Yes will change from react-tooltip to primeReact tooltip

**Comment by eddiejaoude** [comment]
Thank you for the contribution and collaboration 👍

**Pull request merged**